x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -349,25 +342,22 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
       setStartDate(startDateWithZeroAlerts);
```

Comment:
this is probably a dumb question, but aren't putting more load on ElasticSearch by querying 8 years of data? The dates a few lines above are January 1st 2002 to January 1st 2010.
I don't think it would make the test any different if we set those dates to January 1st 2002 to January 2nd 2002 right?

Reply:
makes sense.. Just corrected it.